specs/core/oslc-core.html

```diff
@@ -594,18 +594,18 @@ <h2>Resource Constraints</h2>
               an LDPC for all the shapes for a spec version, at a URI defined by the following pattern:
             </p>
             <pre>
-      http://open-services.net/ns/[vocab short name]/shapes/[SPEC-version]
+      http://open-services.net/ns/[vocab-short-name]/shapes/[SPEC-version]
     </pre
             >
           </div>
           <p>
-            For example, for Change Management 3.0, there should be a container at:
-            <code>http://open-services.net/ns/cm/shapes/3.0</code> with members such as:
+            For example, for Change Management, there should be a container at:
+            <code>http://open-services.net/ns/cm/shapes</code> with members such as:
           </p>
           <pre>
-        http://open-services.net/ns/cm/shapes/3.0/changerequest
-        http://open-services.net/ns/cm/shapes/3.0/somenewshape
-        http://open-services.net/ns/cm/shapes/2.0/unchangedshape
+        http://open-services.net/ns/cm/shapes/3.0#TaskShape
+        http://open-services.net/ns/cm/shapes/3.1#WeaknessShape
```

Comment:
What is version 3.1?

Reply:
Just an example, not intended to mean anything other than to indicate what some future version of CM might look lik.